client/jetpack-connect/main.jsx

```diff
@@ -473,9 +470,7 @@ class JetpackConnectMain extends Component {
 					<div className="jetpack-connect__navigation">{ this.renderBackButton() }</div>
 				</div>
 				<LoggedOutFormLinks>
-					<JetpackConnectHappychatButton eventName="calypso_jpc_instructions_chat_initiated">
-						<HelpButton />
-					</JetpackConnectHappychatButton>
+					<HelpButton />
```

Comment:
This removes the help button from the instructions step. Is that intentional?

Reply:
well, the idea is to remove the button for the whole step, and that's a side-effect of it than anything else. Maybe in the future, we can revisit this to make it a little more complex and select where to show it or not